api/v1beta1/flinkcluster_types.go

```diff
@@ -389,7 +389,7 @@ type FlinkClusterSpec struct {
 
 	// BatchSchedulerName specifies the batch scheduler name for JobManager, TaskManager.
 	// If empty, no batch scheduling is enabled.
-	BatchSchedulerName string `json:"batchSchedulerName,omitempty"`
+	BatchSchedulerName *string `json:"batchSchedulerName,omitempty"`
```

Comment:
After you change the type, you also need to change the places for how this field is used. The test is failing, could you fix it?

Reply:
Sorry, i missed it